### ISSUE TITLE
Remove NOTICES from the functions and change them for debug

### ISSUE THREAD

**Issue by ethervoid** [CartoDB/observatory-extension#73]
If we there is DEBUG trace in the code, which for my point of view it should be there when the functio reach production, should be in a log level minor to notice because it appears in the SQL API for example:

![captura de pantalla 2016-05-16 13 16 49](https://cloud.githubusercontent.com/assets/741240/15288154/af49b482-1b68-11e6-9957-f47e2993da74.png)

**Comment by ohasselblad** [comment]
@ethervoid, is this something we should do before release?

**Comment by ethervoid** [comment]
Not really, but we have to change it in the future and only keep that ones that meaning something for the users
